src/zc/form/field.py

```diff
@@ -420,7 +413,7 @@ def _validate(self, value):
             if len_value != len(self.fields):
                 raise MessageValidationError(
                     _combination_wrong_size_error)
-            for v, f in map(None, value, self.fields):
+            for v, f in zip(value, self.fields):
```

Comment:
And here.

Reply:
The equal length is tested in the preceding if block, so we should be safe here.